src/utils/urls.js

```diff
@@ -65,8 +66,29 @@ export function getSettingsUrl() {
   return settingsPath;
 }
 
-export function getImportUrl() {
-  return importPath;
+export function getImportsUrl() {
+  return importsPath;
+}
+
+export function getCreateImportUrl(sourceName) {
+  const url = new URL(document.location);
+  const keys = [];
+  for (const key of url.searchParams.keys()) {
+    keys.push(key);
+  }
```

Comment:
Not sure I get this part 🤔 Can it be reduced to `const keys = url.searchParams.keys()`?

Reply:
Actually, why does this URL creation need to be so complex?
Would it not be simpler to just compose the URL? 🤷‍♀️

Reply:
I also thought that smth like `${createImportsPath}?source=${encodeURIComponent(sourceName)}` would be sufficient

Reply:
I kinda took this from the creation of the search url, I'm a bit afraid of coming here by a bad link, I want to remove all  keys from the url that don't belong (copy pasta errors can occur). Probably it's just in my head though, no one will get to this page via copy paste :). I'll simplify.

Reply:
Yeah this is a very simple URL, no need to overthink here.

Reply:
Solved